### PR TITLE
Pranav2

### PR DESCRIPTION
**Pull request by Pranavv213**
Book Reader changes made

### CODE REVIEW

File: PA.py [hunk 1]
```diff
@@ -122,7 +128,11 @@ def get_command():
                          "Programs\\Eclipse\\Eclipse IDE for Java Developers - 2020-06")
 
         elif 'open notepad' in query:
-            os.startfile("C:\\Windows\\notepad.exe")
+
+            os.startfile("C:\\Users\\91954\\AppData\\Roaming\\Microsoft\\Windows\\"
```
Comment (Umesh-01) [inline PA.py:132]:
```suggestion
            os.startfile("C:\\Windows\\notepad.exe")
```

File: PA.py [hunk 2]
```diff
@@ -122,7 +128,11 @@ def get_command():
...
+                         "Start Menu\\Programs\\Accessories\\Notepad")
```
Comment (Umesh-01) [inline PA.py:133]:
```suggestion
```

File: PA.py [hunk 3]
```diff
@@ -122,7 +128,11 @@ def get_command():
                          "Programs\\Eclipse\\Eclipse IDE for Java Developers - 2020-06")
 
         elif 'open notepad' in query:
-            os.startfile("C:\\Windows\\notepad.exe")
+
```
Comment (Umesh-01) [inline PA.py:131]:
```suggestion
```

**Review by Umesh-01 — CHANGES_REQUESTED** [review]

**Comment by Pranavv213** [comment]
This issue is not mine. Opening notepad  is someone else issue. Plz check.
My issue was to add Book Reader feature in PA.py which I have already done and made pull request which has no merge conflicts. PLz see to it and merge my pull request for book reader.

**Comment by Umesh-01** [comment]
> This issue is not mine. Opening notepad is someone else issue. Plz check. My issue was to add Book Reader feature in PA.py which I have already done and made pull request which has no merge conflicts. PLz see to it and merge my pull request for book reader.

Yes @Pranavv213 I know that this change was made in different PR but that PR is already merged. May be you haven't pulled the changes from the main branch that's why it is showing the previous code in your branch. Now, you can make the necessary changes, and then I will merge this PR. 👍

**Review by Umesh-01 — COMMENTED** [review]
@Pranavv213 are you working on this??

**Pull request closed without merging**